docs/units-and-global-variables.rst

```diff
@@ -311,14 +311,17 @@ For more information, see the section on :ref:`address`.
     semantics than ``delegatecall``.
 
 
-.. index:: this, selfdestruct
+.. index:: this, selfdestruct, super
 
 Contract Related
 ----------------
 
 ``this`` (current contract's type)
-    the current contract, explicitly convertible to :ref:`address`
+    The current contract, explicitly convertible to :ref:`address`
 
+``super``
+    The parent contract which the current contract inherits from
+    
```

Comment:
You have some trailing whitespace here. Note that `chk_coding_style` job is failing because of this.